installer/volcano-development.yaml

```diff
@@ -4661,8 +4661,7 @@ spec:
           configMap:
             name: volcano-scheduler-configmap
         - name: klog-sock
```

Comment:
```
          env:
            - name: DEBUG_SOCKET_DIR
              value: /tmp/klog-socks
```
The scheduler deployment will read this path `/tmp/klog-socks` by env? Does it work after this pr?

Reply:
The mount path inside of the container has not been changed, the emptydir will be mounted at `/tmp/klog-socks`.